doc/faq.rst

```diff
@@ -65,28 +65,29 @@ optimized file format such as HDF5 to reduce data load times. Various libraries
 such as H5Py, PyTables and pandas provides a Python interface for reading and
 writing data in that format.
 
-Can I add this new algorithm that I (or someone else) just published?
--------------------------------------------------------------------------
-No. As a rule we only add well-established algorithms. A rule of thumb is at least
-3 years since publications, 200+ citations and wide use and usefullness. A
+What are the inclusion criteria for new algorithms ?
+-----------------------------------------------------
+As a rule we only add well-established algorithms. A rule of thumb is at least
+3 years since publications, 200+ citations and wide use and usefullness. A 
 technique that provides a clear-cut improvement (e.g. an enhanced  data
-structure or efficient approximation) on a widely-used method will also be
-considered for inclusion.
+structure or efficient approximation or both) on a widely-used method will 
+also be considered for inclusion. In general we only accept algorithms that
+fit well within the current API of scikit-learn, that is a ``fit``, 
+``predict/transform`` interface.
+On top of this it is suggested that the contributor demonstrate the 
+usefulness of the proposed algorithm/implementation with research papers, 
+articles, some applications, use-cases and any other evidence to support the
```

Comment:
what's the difference between research papers and articles? What's the difference between applications and use-cases?
Maybe say "research papers and common use-cases"?